typescript/sdk/src/utils.ts

```diff
@@ -53,7 +55,83 @@ export function delay(ms: number): Promise<void> {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
+export class MultiGeneric<Value, Networks extends ChainName = ChainName> {
```

Comment:
Can't say I love this name `MultiGeneric` but I also don't have a better suggestion